java/client/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -37,60 +37,52 @@
  * This class is provided as a convenience for easily testing the Chrome browser. The control server
  * which each instance communicates with will live and die with the instance.
  *
+ * <p>
  * To avoid unnecessarily restarting the ChromeDriver server with each instance, use a
  * {@link RemoteWebDriver} coupled with the desired {@link ChromeDriverService}, which is managed
- * separately. For example: <pre>{@code
+ * separately. For example:
  *
- * import static org.junit.Assert.assertEquals;
- *
- * import org.junit.*;
- * import org.junit.runner.RunWith;
- * import org.junit.runners.JUnit4;
- * import org.openqa.selenium.chrome.ChromeDriverService;
- * import org.openqa.selenium.remote.DesiredCapabilities;
- * import org.openqa.selenium.remote.RemoteWebDriver;
- *
- * {@literal @RunWith(JUnit4.class)}
- * public class ChromeTest extends TestCase {
+ * <pre>
+ * public class ChromeTest {
  *
  *   private static ChromeDriverService service;
  *   private WebDriver driver;
  *
- *   {@literal @BeforeClass}
- *   public static void createAndStartService() {
+ *   &#064;BeforeClass
+ *   public static void createAndStartService() throws IOException {
```

Comment:
It does.
See [DriverService#start](https://github.com/SeleniumHQ/selenium/blob/master/java/client/src/org/openqa/selenium/remote/service/DriverService.java#L155):

    public void start() throws IOException {